src/tests/dbus-tests/run_tests.py

```diff
@@ -252,8 +252,11 @@ def udev_shake():
             os.unlink(disk_file)
 
     # dump cropped journal to log file
-    with open('journaldump.log', "w") as outfile:
-        subprocess.call(['journalctl', '-S', start_time], stdout=outfile)
+    try:
+        with open('journaldump.log', "w") as outfile:
+            subprocess.call(['journalctl', '-S', start_time], stdout=outfile)
+    except Exception as e:
+        print('Failed to save journal: %s' % str(e))
```

Comment:
I think this message should go to the same file as the journal dump would go.

Reply:
Great idea. Fixed.